sdk/search/Azure.Search.Documents/src/Spatial/GeoLineStringProxy.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace Azure.Search.Documents
+{
+    /// <summary>
+    /// Proxy for a <see cref="Azure.Core.GeoJson.GeoLineString"/> class.
```

Comment:
Are you adding proxies (or was this one file just detected as new, but was moved/edited (a lot)? If so, why add new proxies? The idea was to get rid of them once GeoJSON types were added. If this is still experimental, I agree for now this is the right thing to do, but since the `EXPERIMENTAL_SPATIAL` preproc conditions were removed, I'm confused as to what the goal is in this PR.

Reply:
Maybe I misunderstood the purpose of the proxy objects.
I thought they were present to allow unknown objects to be serialized into GeoJson object types wherever possible. Is that not the case?

Are you saying we can remove all the code related to proxy objects?